services/src/main/java/org/exoplatform/task/util/TaskUtil.java

```diff
@@ -776,14 +776,37 @@ public static Event buildEvent(Event event, Task task) {
           event.setPriority(Event.PRIORITY_NONE);
       }   
     }    
-    event.setSummary(encoder.encode(task.getTitle()));
+    event.setSummary(encodeCalendarString(task.getTitle()));
```

Comment:
Did you try to remove completely the encoding ? I did a quick test (with PLF 5.0) , the special characters are already correctly encoded on calendar side, so no need to add extra encoding.